squad/api/rest.py

```diff
@@ -1310,9 +1311,19 @@ class StatusViewSet(NestedViewSetMixin, ModelViewSet):
 
 class AttachmentSerializer(serializers.ModelSerializer):
 
+    download_url = serializers.SerializerMethodField()
+
     class Meta:
         model = Attachment
-        fields = ('filename', 'mimetype', 'length')
+        fields = ('download_url', 'filename', 'mimetype', 'length')
```

Comment:
do we really need the 'filename' field here, that is in the download_url now or?

Reply:
Well, it wouldn't hurt I think it's fine IMHO